packages/react/src/experimental/OneDataCollection/visualizations/collection/Table/index.tsx

```diff
@@ -23,6 +23,16 @@ import {
 import { CollectionProps, RecordType } from "../../../types"
 import { useData } from "../../../useData"
 import { useSelectable } from "../../../useSelectable"
+import { PropertyRendererType } from "../../../visualizations/property"
+
+// Define padding values for different property types
+const PROPERTY_TYPE_PADDINGS: Partial<Record<PropertyRendererType, number>> = {
```

Comment:
I think each cell type should provide its own padding, or at least return it to avoid couple the visualization and the cell types code